src/arch/readableTimeString.cpp

```diff
@@ -0,0 +1,19 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Samator Indo Gas
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *  Ketut Kumajaya - initial API and implementation and/or initial documentation
+ *******************************************************************************/
+
+#include "../core/iec61131_functions.h"
+
+std::string getRealtimeString() {
+  std::string time(27, '\0'); // DT#1970-01-01-00:00:00.000
```

Comment:
please use the `getToStringBufferSize()` of `CIEC_DATE_AND_TIME` so that you can be save against any changes in the format.

Reply:
I'm so blind. Fixed.